### DISCUSSION TITLE
Github Strike

### DISCUSSION THREAD

**Discussion by sf9kanu** [community/community#77797] [Other Feature Feedback, Questions, & Ideas]
Hi, everyone. Is there anything I could do with broken strike on github, e.g. change timezone and commit something orlike that?

**Comment by ColinHuelsbusch** [comment]
Hi,

you can use `git commit --date`.

Kind regards,
Colin

**Reply by ColinHuelsbusch** [reply]
Hi, 

please mark my post as answer, if it solved your question. 

Thanks!
Colin
